app/services/importer/get_relevant_job_urls.rb

```diff
@@ -22,9 +22,9 @@ def fetch_jobs
     def return_relevant_jobs
       data = @ats.fetch_company_jobs(@ats_identifier)
       relevant_jobs = []
-      data&.each do |job|
-        title, location = @ats.fetch_title_and_location(job)
-        relevant_jobs << @ats.fetch_url(job) if relevant?(title, location)
+      data&.each do |job_data|
```

Comment:
Just wondering if there is a better way to filter relevant jobs whilst we look at this. @daniel-sussman / @patanj101 - any thoughts on how else we might only draw in relevant jobs from each company's API endpoint?

One thought is that we could just pull all the jobs at this point and then add the filter to the query later. I'd be interested in what effect this had on performance - you don't have to check whether each job is relevant, but you do have to import more jobs.